app/assets/stylesheets/fonts.scss

```diff
@@ -1,19 +1,43 @@
 $brand: "Comfortaa", "Helvetica", sans-serif;
+$body-text: "Fira Sans", "Helvetica", sans-serif;
 
 .monospace{
   font-family: "Fira Mono", monospace;
   font-size: 1.1rem;
 }
 
+.brand {
+  font-family: $brand;
+}
+
+
+
 html {
-  font-family: "Fira Sans", "Helvetica", sans-serif;
+  font-family: $body-text;
 }
 
-.brand {
-  font-family: $brand;
+h1 {
+  @extend .brand;
+  text-align: center;
+}
+h2 {
+  @extend .dark-text;
+}
+h3 {
+  @extend .dark-text;
+}
+h4{
+  @extend .dark-text;
+}
+h5{
+
+}
+h6{
+  
 }
 
 p {
+  @extend .dark-text.secondary;
```

Comment:
yup, you're right `color: rgba(0, 0, 0, 0.87)` is a better option. I'll make the changes for this.

The material design document isn't entirely clear what it meant by primary/secondary text, ie the title is primary and paragraphs secondary or title and paragraphs are both primary and something lower than that secondary.